packages/streaming-image-volume-loader/src/helpers/makeVolumeMetadata.ts

```diff
@@ -28,8 +29,10 @@ export default function makeVolumeMetadata(
   const voiLutModule = metaData.get('voiLutModule', imageId0);
 
   // voiLutModule is not always present
+  let voiLUTFunction;
   if (voiLutModule) {
     const { windowWidth, windowCenter } = voiLutModule;
+    voiLUTFunction = voiLutModule?.voiLutModule;
```

Comment:
I think this should be `voiLutModule?.voiLUTFunction `

Reply:
yes i think that was my mistake